transaction.js

```diff
@@ -10,6 +10,7 @@ var MessageStream = require('./messagestream');
 var trans = exports;
 
 var MAX_HEADER_LINES = config.get('max_header_lines') || 1000;
+var REMOVE_CARRIAGE_RETURN = config.get('remove_carriage_return') || 'true';
```

Comment:
This is what the code in transaction.js might look like, with the boolean declaration in smtp.ini:

```diff
diff --git a/transaction.js b/transaction.js
index 75cd382..9933480 100644
--- a/transaction.js
+++ b/transaction.js
@@ -46,7 +46,11 @@ exports.createTransaction = function(uuid) {
     t.uuid = uuid || utils.uuid();
     // Initialize MessageStream here to pass in the UUID
     t.message_stream = new MessageStream(
-            config.get('smtp.ini'), t.uuid, t.header.header_list);
+        config.get('smtp.ini', {
+            booleans: [
+                '+main.strip_crlf',
+            ]
+        }), t.uuid, t.header.header_list);
     return t;
 };
 
```

The leading + is what defines the boolean as true, unless it has been declared false in the config file.